cfme/services/catalogs/catalog_item.py

```diff
@@ -260,29 +232,37 @@ def create(self):
         sel.click(template_select_form.add_button)
 
     def update(self, updates):
-        catalog = fakeobject_or_object(self.catalog, "name", "<Unassigned>")
-        sel.force_navigate('catalog_item_edit',
-                           context={'catalog': catalog.name,
-                                    'catalog_item': self})
+        fakeobject_or_object(self.catalog, "name", "<Unassigned>")
```

Comment:
this just returns object or fake object that was used earlier. Now it is not needed.